Source/Core/AudioCommon/Mixer.h

```diff
@@ -51,6 +51,10 @@ class Mixer final
   float GetCurrentSpeed() const { return m_speed.load(); }
   void UpdateSpeed(float val) { m_speed.store(val); }
 
+  static constexpr u64 FIXED_SAMPLE_RATE_DIVIDEND =
+      54000000 *
+      2;  // 54000000 doesn't work here as it doesn't evenly divide with 32000, but 108000000 does
```

Comment:
Please put the comment on the line above the variable definition. It will make the formatting neater.